challenges/fifo_animal_shelter/fifo_animal_shelter.py

```diff
@@ -0,0 +1,68 @@
+class AnimalShelter():
+
+    def __init__(self):
+        self.headCat = None
+        self.headDog = None
+        self.tailCat = None
+        self.tailDog = None
+        self.animalAge = 0
+
+    """ enqueues """
+    def enqueue(self, animalName, animalType):
```

Comment:
According to the spec, `.enqueue()` takes one argument that is a Cat or Dog object.